contrib/storage-druid/src/main/java/org/apache/drill/exec/store/druid/common/DruidAndFilter.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.store.druid.common;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DruidAndFilter extends DruidFilterBase {
+
+  private String type = DruidCompareOp.AND.getCompareOp();
+  private List<String> fields = new ArrayList<>();
+
+  public DruidAndFilter(@JsonProperty("fields") List<String> fields) {
+    this.fields = fields;
+  }
+
+  public String getType() {
+    return type;
+  }
+
+  public void setType(String type) {
+    this.type = type;
+  }
```

Comment:
For "data classes" such as this, it is often handy to make them immutable: fields are `final` and the two values are passed into the constructor.

Or, since this one seems to always use that one Druid function, just do:
```
  public String getType() {
    return DruidCompareOp.AND.getCompareOp();
  }
```